examples/context/webpack.config.js

```diff
@@ -1,32 +1,23 @@
-/* eslint-disable import/extensions */
-import path from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import webpack from 'webpack';
 import HtmlWebpackPlugin from 'html-webpack-plugin';
 
 // This config doesn't have entry and output set up because it's not meant to
-// work standalone. Cosmos adds an entry & output when extending this.
+// work standalone. react-cosmos-webpack adds an entry & output when extending this.
 module.exports = {
   devtool: 'eval',
   resolve: {
-    extensions: ['', '.js', '.jsx'],
-  },
-  resolveLoader: {
-    // This can be removed in a real life scenario. react-cosmos-webpack is a
-    // sibling directory in this setup, whereas a regular npm module is nested
-    // within the app. In the latter case, node modules installed in the host
-    // app are found by react-cosmos-webpack due to Node's bubble up require
-    // strategy.
-    root: path.join(__dirname, 'node_modules'),
+    extensions: ['.js', '.jsx'],
   },
   module: {
     loaders: [{
       test: /\.jsx?$/,
-      loader: 'babel',
+      loader: 'babel-loader',
       exclude: /node_modules/,
     }],
   },
   plugins: [
-    new webpack.NoErrorsPlugin(),
+    new webpack.NoEmitOnErrorsPlugin(),
```

Comment:
I guess I didn't delete the root `node_modules` before I started working and still had webpack version 1.14 installed. If you perform a clean install it should be fine.